qa/tasks/deepsea.py

```diff
@@ -385,6 +390,26 @@ def _maybe_apply_alternative_defaults(self):
                     )
         dump_file_that_might_not_exist(self.master_remote, global_yml)
 
+    def _maybe_apply_netwrok(self):
+        network_cfg_dir = '/srv/pillar/ceph/stack/ceph'
+        network_cfg = network_cfg_dir + '/cluster.yml'
+
+        if self.cluster_network or self.public_network:
+            self.master_remote.run(
+                args='sudo mkdir -p %s && ' % network_cfg_dir +
+                'sudo touch %s && ' % network_cfg +
+                'sudo chown salt.salt %s -R' % network_cfg_dir)
```

Comment:
There is more sense to use single multiline string with format, like:
```
self.master_remote.sh("sudo mkdir -p {dir} &&"
                      "sudo touch {cfg} &&"
                      "sudo chown salt.salt {dir} -R"
                      .format(dir=network_cfg_dir, cfg=network_cfg))
```

Reply:
fixed